### ISSUE TITLE
Could you please tell me which part of the code implementation corresponds to the Dual-views Kullback-Leibler divergence?

### ISSUE THREAD

**Issue by gangxu822** [Cliffia123/LegoGCD#2]

**Comment by Cliffia123** [comment]
Hi @gangxu822 , you can find the Dual-views Kullback-Leibler divergence in `model.py` from lines 236 to 241.
```
student_out_1, student_out_2 = student_out
student_out_1 = torch.softmax(student_out_1, dim=-1)
student_out_2 = torch.softmax(student_out_2, dim=-1)

loss_entropy = torch.sum(student_out_1 * (F.log_softmax(student_out_1, dim=-1)-F.log_softmax(student_out_2, dim=-1)), dim=-1)
total_loss += loss_entropy.mean()
